combat-trainer.lic

```diff
@@ -1055,6 +1094,36 @@ class SpellProcess
     game_state.casting = false
   end
 
+  def check_regalia(game_state)
+    game_state.swap_regalia_type = nil if game_state.swap_regalia_type == game_state.last_regalia_type
+    return if game_state.casting
+    return unless game_state.swap_regalia_type
+    if @regalia_spell.nil?
```

Comment:
❓ What do you think about moving this so it only happens once? Or moving it to `validate.lic`? It feels pretty similar to the checks on line 38 (though it should belong to a different Process)

Reply:
I'll gladly add some stuff to validate.  I'd say most of your suggestions would probably work best over there.

Reply:
Now only checks once during initialize.  Also removed the warning and moved a check for it into validate.lic